src/pages/index.tsx

```diff
@@ -1,29 +1,48 @@
 import CampaignHero from "../components/hero/campaignHero";
-import { featuredImageID } from "../utils/utils";
-import {
-  SkipNavigation,
-  TemplateAppContainer,
-} from "@nypl/design-system-react-components";
+import SwimLanes from "../components/swimlanes/swimLanes";
+import { TemplateAppContainer } from "@nypl/design-system-react-components";
+import data from "@/data/lanes";
+import { getNumItems, featuredImageID } from "@/utils/utils";
 
 export default function Home(props: any) {
   return (
     <TemplateAppContainer
       aboveHeader={<p> Notification banner </p>}
       header={<p> Header </p>}
       breakout={<CampaignHero featuredImageID={props.featuredImageID} />}
-      contentPrimary={
-        <>
-          <p>First swim lane</p>
-          <p>Featured Content</p>
-          <p>Rest of swim lanes</p>
-          <p>Explore further links</p>
-        </>
-      }
+      contentPrimary={<SwimLanes lanesWithNumItems={props.lanesWithNumItems} />}
       renderSkipNavigation={true}
     />
   );
 }
 
 export async function getServerSideProps() {
-  return { props: { featuredImageID: featuredImageID() } };
+  const lanes = data.lanes;
+  const flatCollections = [].concat(...lanes.map((lane) => lane.collections));
+  const collectionsWithNumItems = await Promise.allSettled(
+    flatCollections.map(async (collection) => {
+      try {
+        const numItems = await getNumItems(collection.uuid);
+        return { ...collection, numItems };
+      } catch (error) {
+        return { ...collection, numItems: 0 };
+      }
+    })
+  );
+  const updatedLanes = lanes.map((lane) => {
+    const updatedCollections = lane.collections.map(() => {
+      const result = collectionsWithNumItems.shift();
+      return result.status === "fulfilled"
+        ? result.value
+        : { ...result, value: 0 };
```

Comment:
`value` might be better as an empty object since the successful result contains `uuid`, `title`, etc. Otherwise, you may get errors attempting to make a reference such as `collection.uuid` when the returned result is `0`.